.github/CONTRIBUTORS.MD

```diff
@@ -13,7 +13,7 @@ Specials thanks to the members of Stevens Blueprint for collaborating on this pr
 - [Aidan Ouckama](https://github.com/ouckah)
 - [Diego Ramirez](https://github.com/diego-developed)
 - [Joanne Wang](https://https://github.com/joanne1229)
-- [Lucas Ha](https://github.com/Lucasha11)
+- [Lucas Ha](https://github.com/Lucasha11)  
```

Comment:
We try to make pull requests as atomic as possible, meaning PRs should only contain changes made to fix the bug/add new features. Space/formatting changes should be part of another PR.